package.json

```diff
@@ -6,7 +6,7 @@
     "./apps/*"
   ],
   "scripts": {
-    "build": "npm run build --workspaces --if-present"
+    "build": "exit 1"
```

Comment:
This change isn't necessary just added to verify the root `build` script isn't being run.